docs/cookbook/setting_up_amazon_aws_s3_clustering.md

```diff
@@ -0,0 +1,83 @@
+# Setting up Amazon AWS S3 clustering
+
+When setting up clustering, you can use Amazon AWS S3 as a binary handler,
+meaning it will be used to store binary files.
```

Comment:
it => AWS S3

To distinguish between flysystem aws s3 handler and the AWS S3 service itself :P